Week-1/HTML Assignment1/index.html

```diff
@@ -0,0 +1,153 @@
+         border-collapse: collapse;
+       }
+       td {
+            text-align: left;
+            padding: 10px 130px;
+        }
+        tr:nth-child(even) {
+          background-color: #B8B8B8;
+            padding-left: 20px;
+        }
+    </style>
+  </head>
+  <body>
+    <main>
+    <nav class="vertical" style ="float: left">
+      <img class ="verIcon" src="icon/bars-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/home-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/search-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/volume-up-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/undo-alt-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/cog-solid.svg" width="12" height="13"></img></br>
+      <img class ="verIcon" src="icon/soundcloud.svg" width="12" height="13"></img>
+    </nav>
+    <div style ="background-color:#505050 ">
+    <div style="display:inline-block; padding: 20px 20px 20px">
+      <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+      width="120"
+      height="130">
+    </div>
+    <div style ="padding-left: 10px ; display: inline-block;">
+        <b style = "font-size: large; color: white">CattyBoard Top 100 Single Charts(11.06.36)</b>
+        </br>
+        <b class="boldh">Unknown Artist</b>
+        </br>
+        <b class="boldh">2016.Charts.100 songs</b>
+        </br></br></br></br>
+        <img src="icon/play-solid.svg"  width="12" height="13">
+        <b class="iconWithLabel">Play All</b>
+        <img src="icon/plus-solid.svg" width="12" height="13"></img>
+        <b class="iconWithLabel">Add to</b>
+        <b class="iconWithLabel">... More</b>
+    </div>
+    </div> 
+	<table>
+        <tr>
+          <td>1.One Dance</td>
+          <td>Crake feat CatKid & Cyla</td>
+          <td>2.54</td>
+          <td><label class="CATTY">CATTY CLOUD SYNC</label></td>
+        </tr>
+        <tr>
+            <td>2.Panda</td>
+            <td>Cattee</td>
+            <td>4.06</td>
+            <td><label class="CATTY">CATTY CLOUD SYNC</label></td>
+        </tr>
+        <tr>
+            <td>3.Can't Stop the Feeling!</td>
+            <td>Catin Cimberlake</td>
+            <td>3.56</td>
+            <td><label class="CATTY">CATTY CLOUD SYNC</label></td>
+        </tr>
+        <tr>
+            <td>4.Work From Home</td>
+            <td>Cat Harmony feat Colla</td>
+            <td>3.34</td>
+            <td><label class="CATTY">CATTY CLOUD SYNC</label></td>
+        </tr>
+      </table>
+    </main> 
+     <footer class="footerSection">
+      <div style="background-color: #6082b6; bottom: 1px; left: 5px">
+        <div style="float: left;">
+          <img
+            src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+            width="80"
+            height="80"
+            style="padding: 10px 10px 5px;"
```

Comment:
do not use inline styles move it to external css file

Reply:
moved all css into external